app/javascript/topNavigation/utilities.js

```diff
@@ -54,12 +55,12 @@ export function initializeMemberMenu(memberTopMenu, menuNavButton) {
   }
   const { classList } = memberTopMenu;
   menuNavButton.addEventListener('click', (_event) => {
-    if (classList.contains('showing') && classList.contains('clicked')) {
+    if (classList.contains('showing') && memberTopMenu.dataset.clicked) {
       closeHeaderMenu(memberTopMenu, menuNavButton);
       menuNavButton.focus();
     } else {
       openHeaderMenu(memberTopMenu, menuNavButton);
-      classList.add('clicked');
+      memberTopMenu.setAttribute('data-clicked', 'clicked');
```

Comment:
Refactor (non-blocking): Use the `dataset` object to set the `data-clicked` attribute.

```suggestion
      memberTopMenu.dataset.clicked = 'clicked';
```

Reply:
That is actually much nicer. Thanks for recommending.